fireball.js

```diff
@@ -71,6 +72,33 @@ if (Meteor.isClient) {
       start();
     }
   });
+
+  Template.food.helpers({
+    food: function () {
+      return Session.get("food");
+    }
+  });
+
+  Template.food.events({
+    'click button': function () {
+      var choice = Math.floor((Math.random() *3) +1);
+      if (choice == 1) {
+        Session.set("food","People");
+        Session.set("counter", Session.get("counter")-3);
+        alert("We don't like eating people -3 points");
```

Comment:
We can change the value to something a little bit more appealing.  We can change the alert message as well, but the alert seems to be getting in the way of the game.